recipe/recipe.yaml

```diff
@@ -11,19 +11,30 @@ source:
   sha256: d7d8a58878c037372022f0c1d8fa58e15eb3d198498b521332d05def84cff351
 
 build:
-  number: 0
+  number: 1
   skip:
     - win
   script:
     - go-licenses save --save_path $SRC_DIR/library_licenses .
-    - go build -ldflags="-s -w -X github.com/brevdev/brev-cli/brev-cli.Version=${{ version }}" -v -o $PREFIX/bin/brev
-    # Install bash, zsh, and fish completions 
-    - mkdir -p $PREFIX/share/bash-completion/completions
-    - brev completion bash > $PREFIX/share/bash-completion/completions/brev
-    - mkdir -p $PREFIX/share/zsh/site-functions
-    - brev completion zsh > $PREFIX/share/zsh/site-functions/_brev
-    - mkdir -p $PREFIX/share/fish/vendor_completions.d
-    - brev completion fish > $PREFIX/share/fish/vendor_completions.d/brev.fish
+    - GOOS="${GOOS}" GOARCH="${GOARCH}" go build -ldflags="-s -w -X github.com/brevdev/brev-cli/brev-cli.Version=${{ version }}" -v -o ${PREFIX}/bin/brev
+    # Default to no cross-compile
+    - export BREV="${PREFIX}/bin/brev"
+    - if: unix and target_platform != build_platform
+      then:
+        # Need to compile for the build_platform archi
+        - if: build_platform == "linux-64" or build_platform == "osx-64"
+          # Fail if build_platform changes in future to non-x86_64 arch
+          then: export GOARCH=amd64
+        - GOOS="${GOOS}" GOARCH="${GOARCH} go build -ldflags="-s -w -X github.com/brevdev/brev-cli/brev-cli.Version=${{ version }}" -v -o ${BUILD_PREFIX}/bin/brev
+        - export BREV="${BUILD_PREFIX}/bin/brev"
```

Comment:
I refactored the logic a bit to (hopefully) make decisions more clear, but let me know if not.